src/main/java/racingcar/strategy/RandomMovingStrategy.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.strategy;
+
+import java.util.Random;
+
+public class RandomMovingStrategy implements MovingStrategy {
+
+  private final int MOVING_REFERENCE_VALUE = 4;
+  private final int BOUND_LIMIT = 10;
+  @Override
+  public boolean isMovable() {
+    if (MOVING_REFERENCE_VALUE <= getRandomNumber()) {
+      return true;
+    }
+    return false;
```

Comment:
```suggestion
    return MOVING_REFERENCE_VALUE <= getRandomNumber();
```

좀 더 간단하게 표현이 가능해요 :)